prometheus-openstack-exporter

```diff
@@ -611,29 +611,34 @@ class Nova:
             elif type(cpu_info) != dict:
                 cpu_info = json.loads(cpu_info)
                 arch = cpu_info["arch"]
-            label_values = [
-                config["cloud"],
-                host,
-                self.aggregate_map.get(host, "unknown"),
-                self.services_map[host],
-                arch,
-            ]
-            # Disabled hypervisors return None below, convert to 0
-            vms.labels(*label_values).set(squashnone(h["running_vms"]))
-            vcpus_total.labels(*label_values).set(squashnone(h["vcpus"]))
-            vcpus_used.labels(*label_values).set(squashnone(h["vcpus_used"]))
-            mem_total.labels(*label_values).set(squashnone(h["memory_mb"]))
-            mem_used.labels(*label_values).set(squashnone(h["memory_mb_used"]))
-            disk_total.labels(*label_values).set(squashnone(h["local_gb"]))
-            disk_used.labels(*label_values).set(squashnone(h["local_gb_used"]))
-
-            if config.get("schedulable_instance_size", False):
-                schedulable_instances.labels(*label_values).set(
-                    self._get_schedulable_instances(h)
-                )
-                schedulable_instances_capacity.labels(*label_values).set(
-                    self._get_schedulable_instances_capacity(h)
-                )
+
+            for agg in self.prodstack["aggregates"]:
+                if self.aggregate_map.get(host+agg["name"]) is None:
+                    continue
+                label_values = [
+                    config["cloud"],
+                    host,
+                    self.aggregate_map.get(host+agg["name"], "unknown"),
```

Comment:
**nitpick (non-blocking):** 

"unknown" is not needed any more since you have None value check?

Reply:
done